client/src/components/ActiveChat/ActiveChat.tsx

```diff
@@ -18,7 +18,11 @@ export default function Dashboard(): JSX.Element {
   const classes = useStyles();
   const { convo } = useConvoContext();
   const { loggedInUser } = useAuth();
-  const [otherUser, setOtherUser] = useState<OtherUser>();
+  const [otherUser, setOtherUser] = useState<OtherUser | undefined>({
```

Comment:
If you have a mock user object as default state, why do we also have undefined as a type? :)

Reply:
This is fixed.